resources/db/migration_mariadb/V13__index_management.sql

```diff
@@ -0,0 +1,24 @@
+DROP INDEX IF EXISTS transaction_recipient_id_amount_height_idx ON transaction;
+DROP INDEX IF EXISTS transaction_recipient_id_sender_id_idx ON transaction;
+DROP INDEX IF EXISTS tx_sender_type ON transaction;
+DROP INDEX IF EXISTS tx_block_id_index ON transaction;
+DROP INDEX IF EXISTS block_generator_id_idx ON block;
+DROP INDEX IF EXISTS subscription_time_next_index ON subscription;
+DROP INDEX IF EXISTS subscription_latest_index ON subscription;
+DROP INDEX IF EXISTS subscription_id_latest_idx ON subscription;
+DROP INDEX IF EXISTS account_asset_quantity_idx ON account_asset;
+DROP INDEX IF EXISTS account_asset_assetid_latest_idx ON account_asset;
+DROP INDEX IF EXISTS at_latest_idx ON at;
+DROP INDEX IF EXISTS at_state_id_latest_idx ON at_state;
+CREATE INDEX IF NOT EXISTS reward_recip_assign_acc_latest_height_idx ON reward_recip_assign (account_id, latest, height);
+CREATE INDEX IF NOT EXISTS at_id_latest_idx ON at (id, latest);
+CREATE INDEX IF NOT EXISTS at_state_latest_at_id_idx ON at_state (latest, at_id);
+CREATE INDEX IF NOT EXISTS account_balance_latest_balance_idx ON account_balance (latest, balance);
+CREATE INDEX IF NOT EXISTS account_asset_asset_latest_quantity_account_idx  ON account_asset (asset_id, latest, quantity DESC, account_id ASC);
+CREATE INDEX IF NOT EXISTS account_asset_id_id_latest ON account_asset (account_id, asset_id, latest);
+CREATE INDEX IF NOT EXISTS transaction_recipient_height_id_amount_idx ON transaction (recipient_id, height, id, amount );
+CREATE INDEX IF NOT EXISTS transaction_refhash_type_subtype_height_idx ON transaction (referenced_transaction_fullhash, type, subtype, height);
+CREATE INDEX IF NOT EXISTS transaction_height_id_idx ON transaction (height, id);
+CREATE INDEX IF NOT EXISTS transaction_sender_type_subtype_height_idx ON transaction (sender_id, type, subtype, height);
+CREATE INDEX IF NOT EXISTS subscription_latest_next_id_idx ON subscription (latest, time_next, id);
+CREATE INDEX IF NOT EXISTS subscription_id_latest_idx ON subscription (id, latest);
```

Comment:
so, mariadb drops and creates 12 indices, while PSQL and SQLITE drops and creates 13. 

Also, just cross checking: 
Why are existing indices dropped and recreated?

Example:

```
DROP INDEX IF EXISTS account_asset_assetid_latest_idx ON account_asset
//...
CREATE INDEX IF NOT EXISTS account_asset_id_id_latest ON account_asset (account_id, asset_id, latest);
```
This index does exist already
<img width="941" alt="image" src="https://github.com/user-attachments/assets/b63c03f6-0182-4447-82cd-36a37d171361" />

Reply:
Yes but the order of the columns within the index were not perfect for tbe DB engine to optimize the SQL-Query

Reply:
And yes i dropped one more index for PSQ and SQLite as the naming convention starting wihh a number was not correct - now all indexes are aligned in naming

Reply:
> Yes, but the order of the columns within the index was not perfect for the DB engine to optimise the SQL query.

I checked again the _DROP INDEX IF EXISTS account_asset_assetid_latest_idx ON account_asset_ statement to the new index _CREATE INDEX IF NOT EXISTS account_asset_id_id_latest ON account_asset (account_id, asset_id, latest);_

It seems the columns are the same, so I changed the naming of the index only. 
As I did it for all 3 DBs, it is consistent for all of them.